test/issues/0470.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var assert = require('assert');
+var yaml = require('../..');
+
+test('should not unnecessaryly apply quotes', function () {
+
+  var expected = 'url: https://github.com/nodeca/js-yaml\n';
+  var actual = yaml.dump(
+    {
+      url: 'https://github.com/nodeca/js-yaml'
+    }
+  );
+
+  assert.strictEqual(actual, expected);
```

Comment:
That's not enougth. Need to check that after load back result will be exact as initial